input/2D/relaxation/distributed/grains16_seed456.txt

```diff
@@ -0,0 +1,17 @@
+x                    y
+3211.289878400778434 6772.225174745160075
```

Comment:
How did you generate grainsxx.txt files? Put some instructions?

Reply:
Added comment in the input file.